playwright.config.ts

```diff
@@ -0,0 +1,19 @@
+import { defineConfig, devices } from '@playwright/test';
+
+export default defineConfig({
+    testDir: './tests',
+    timeout: 30 * 1000,
+    expect: {
+        timeout: 5000
+    },
+    use: {
+        baseURL: 'https://dugong.online',
```

Comment:
And here is the most important, you are not going to run the test again version in production.
But we should consider it for release deployment script (which we don't have yet).